react-challenge.html

```diff
@@ -35,20 +35,20 @@
 
         handleMoodOptions = () => {
           let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          this.setState({ currentMood: this.state.moods[randomIndex] })
         }
 
         render(){
           return(
               <h1>Welcome to the Mood Meter App</h1>
 
-              <h3>Mood Options:</h3>
+              <<<h3>>Mood Options:<<</h3>>>
 
               <div style={{ fontSize: "40px", height: "50px"}}>
                 { this.state.currentMood }
               </div>
 
-              <button onClick={ handleMoodOptions }>
+              <button onClick = { this.handleMoodOptions }>
                 Click Here For Next Mood option
               </button>
 
```

Comment:
Looks like you got most of the errors, but you missed the wrapping div bug. Also, you wrapped the h3 in triple arrows, i'm curious as to why.